Server/BackEndMonitoringServer/src/CServiceHandler.cpp

```diff
@@ -0,0 +1,287 @@
+#include "stdafx.h"
+
+#include "CServiceHandler.h"
+#include "Utils.h"
+
+class ServiceHandle
```

Comment:
Don't forget to decide what you have to do with move and copy ctors in future. For example, you can safely delete copy and leave default move ctors in this class.

Reply:
Fixed too.